### PR TITLE
Create API  to get insight datas from insightstore

### PR DESCRIPTION
**Pull request by sanposhiho**
**What this PR does / why we need it**:

create api to get insight data from insight store

**Which issue(s) this PR fixes**:

Ref #1142 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/api/grpcapi/web_api.go [hunk 1]
```diff
@@ -1312,190 +1316,30 @@ func (a *WebAPI) GetInsightData(ctx context.Context, req *webservice.GetInsightD
...
-			Value:    model.DeploymentStatus_DEPLOYMENT_FAILURE,
-		},
-	}
-
-	pageSize := 50
-	successDeployments, err := a.deploymentStore.ListDeployments(ctx, datastore.ListOptions{
-		PageSize: pageSize,
-		Filters:  append(filterForSuccessDeploy, commonFilters...),
-	})
+	chunks, err := a.insightstore.LoadChunks(ctx, claims.Role.ProjectId, req.ApplicationId, req.MetricsKind, req.Step, from, count)
```
Comment (nghialv) [inline pkg/app/api/grpcapi/web_api.go:1322]: Can you add a TODO (before this line) about caching the insight data inside the cache service?

Comment (sanposhiho) [reply pkg/app/api/grpcapi/web_api.go:1322]: OK 👍

**Pull request merged**